data/korath/korath variants.txt

```diff
@@ -857,7 +857,30 @@ ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Jump)"
 		"Steering (Planetary Class)"
 		"Jump Drive"
 
+ship "Kas'lor Ik 582" "Kas'lor Ik 582 (Crippled)"
+	outfits
+		"Langrage Hyper-Heaver" 3
+		"Warder Anti-Missile" 2
+		
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Liquid Sodium Cooler"
+		"Fuel Processor" 5
+		"Microbot Defense Station" 6
+		"Thermal Repeater Rifle" 551
+		
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Jump Drive (Broken)"
```

Comment:
Generally, the `Crippled` variants of Korath ships are missing more than just their Jump Drive.
Usually, they lack any systems core.
At the stage where this mission is offered, I don't think it's unreasonable to expect the player to be able to take on a world ship with its large systems core, but perhaps this variant should be renamed, perhaps `Stranded` to differentiate it from the usual `Crippled` ships.